rpc-tests/tests/0001-register-xcasset.js

```diff
@@ -0,0 +1,38 @@
+async function run(nodeName, networkInfo, args) {
+    networkInfo.nodesByName["astar"].hrmpTo = 2007
+    networkInfo.nodesByName["shiden"].hrmpTo = 2006
+
+    const { sendTransaction } = await import("./tx-utils.mjs");
+    const { wsUri, userDefinedTypes, hrmpTo } = networkInfo.nodesByName[nodeName];
+    const api = await zombie.connect(wsUri, userDefinedTypes);
+
+    await zombie.util.cryptoWaitReady();
+
+    // account to submit tx
+    const keyring = new zombie.Keyring({ type: "sr25519" });
+    const sender = keyring.addFromUri("//" + args[0]);
+
+    const tx0 = await api.tx.assets.forceCreate(hrmpTo, {id: sender.address}, true, 1);
+
+    const assetLocation = `{"v1":{"parents":1,"interior":{"x1":{"parachain":${hrmpTo}}}}}`;
```

Comment:
better to move to v2

Reply:
I can't even see V2 in the menus on any of our live chains.  V2 is enabled from what version on?